book/03-git-branching/sections/rebasing.asc

```diff
@@ -151,27 +151,27 @@ Suppose you clone from a central server and then do some work off that.
 Your commit history looks like this:
 
 .Clone a repository, and base some work on it
-image::images/perils-of-rebasing-1.png["Clone a repository, and base some work on it."]
+image::images/perils-of-rebasing-1.png[Clone a repository, and base some work on it]
```

Comment:
I see you're removing the quotes here. IIRC, this was done because we had punctuation inside the square brackets, and that was wrecking rendering. Do we know this will still work?

Reply:
> Do we know this will still work?    

Yes, it still works. I've build a latest version of book based on my master branch, see below screenshot.

![quote-in-progit](https://user-images.githubusercontent.com/28824347/85912862-42a79f80-b81f-11ea-9586-00e22adac697.png)

> this was done because we had punctuation inside the square brackets, and that was wrecking rendering 

Then perhaps, we should drop the changes. If it had created problem earlier, it'll likely to create problem in future.

Reply:
Accordingly to AsciiDoc format, the comma is a separator for parameters set, for example:
```
image::sunset.jpg[Sunset,300,200]
```
Where `300` and `200` are the width and height of the image.

Originally, the problem is related to **the PDF version** of the book and it's **still actual**.

Reply:
I'll drop it altogether with https://github.com/progit/progit2/pull/1444/commits/d39824ab56d10fa031a985a7dd1fb953947e1ad9 commit.